data/json/snippets/zombie_anatomy.json

```diff
@@ -0,0 +1,89 @@
+      "The layout and segmentation of its bony carapace disturbingly resembles what you want to call plate mail, but its closer to the armor of an arthropod",
+      "These plates are almost impossible to remove until you realize they are linked to the internal skeleton with calcified pillars you must break before removing its armor"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_acid_general>",
+    "text": [
+      "The sheer pungent acrid odor of the carcass makes you temporarily recoil and reconsider the safety of delving into its entrails",
+      "The sputters of acid on your gear you earned by digging into this corpse could maybe warrant the use of some water to douse",
+      "Most cavities in its anatomy seem to have been filled with an acidic solution mixed with a foul black liquid and pieces of its slowly melting tissues",
+      "You are almost certain that its flesh should not be able to contain such a virulently aggressive liquid, yet, it does",
+      "You have to step back as a gland you inadvertently into seems to perform a neutralization reaction between its contents and the rest of its fluids"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_shadow_general>",
+    "text": [
+      "This corpse seems completely ordinary now, and dissection brings you no further to asserting what gave the creature its peculiar properties",
+      "The lingering shadows surrounding it make your work quite a lot harder as they seem to deliberately grow more opaque around your tool as if they wished to hide the monster's secrets",
+      "You spot a few pores still weakly sputtering thin puffs of black smoke, investigating them further reveals glands full of fuming black liquid linked to the pores with long tubes",
+      "You make one incision and the resulting cloud of black smoke forces you to take a short break as it becomes impossible to see anything",
+      "Despite it not being very evident from the outside, it seems the entire digestive tract of the abomination was bloated with its obscuring smoke"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<zombie_harvest_hulk_general>",
+    "text": [
+      "The sheer scale of the carcass leaves you only with a luxury of choice on where to start your work",
+      "Its legs seem to have been reinforced with a strange lattice of skeletal bracing and muscle fibers to support its immense bulk",
+      "It seems most of the creature was grown as some sort of suit looking as if it was piloted by the still normal sized human corpse found in a fetal position in its torso",
+      "The overall anatomy of this enormous monstrosity leaves you only to wonder how it possibly could achieve the range of motion you witnessed while it was still standing",
+      "You cut into its abdomen, and discover its utter lack of internal organs, the expected cavity replaced by nothing but more flesh and muscle"
```

Comment:
```suggestion
      "The sheer scale of the carcass leaves you only with a luxury of choice on where to start your work.",
      "Its legs seem to have been reinforced with a strange lattice of skeletal bracing and muscle fibers to support its immense bulk.",
      "It seems most of the creature was grown as some sort of suit looking as if it was piloted by the still normal sized human corpse found in a fetal position in its torso.",
      "The overall anatomy of this enormous monstrosity leaves you only to wonder how it possibly could achieve the range of motion you witnessed while it was still standing.",
      "You cut into its abdomen, and discover its utter lack of internal organs, the expected cavity replaced by nothing but more flesh and muscle."
```

Reply:
No end punctuation needed.

Reply:
Got feedback from Venera, adding all your suggestions then removing punctuation from the composite snippets.